.env.example

```
@@ -1,7 +1,15 @@
 REDIS_PORT=6379
-# REDIS_PASSWORD=password_if_needed
\ No newline at end of file
+# REDIS_PASSWORD=password_if_needed
+
+# Winston Logging Configuration
+# Options: error, warn, info, http, debug
+LOG_LEVEL=debug
+# Directory for log files   
+LOG_DIR=logs
+# Log detailed metadata (query params, variables) - set to false in production for privacy
+LOG_DETAILED_METADATA=true         
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)